recipes/glimpse-bio/meta.yaml

```diff
@@ -32,12 +31,10 @@ requirements:
 
 test:
   commands:
-    - GLIMPSE_chunk --help
-    - GLIMPSE_concordance --help
-    - GLIMPSE_ligate --help
-    - GLIMPSE_phase --help
-    - GLIMPSE_sample --help
-    - GLIMPSE_snparray --help
+    - GLIMPSE2_chunk --help
+    - GLIMPSE2_split_reference --help
+    - GLIMPSE2_ligate --help
+    - GLIMPSE2_phase --help
```

Comment:
can you comment if some of these commands were removed in 2.0?

Reply:
Those are the new function available in 2.0, I deleted the one that were present in the first version and absent in the second.